Docs update 07 05 (#85)
globalid/dev-tooling-examples: README.md

# GlobaliD Developer Tooling Examples

This repository contains Node.js examples that utilize GlobaliD's developer tooling. It is a monorepo which contains separate sample projects.

> ## Sample apps
> - **[globalid-connect](https://github.com/globalid/dev-tooling-examples/tree/main/globalid-connect)**: Demonstrates login flow using GlobaliD
> - **[verifier-agent](https://github.com/globalid/dev-tooling-examples/tree/main/verifier-agent)**: Demonstrates verification flow for validating a user is 18+

Reviewer: ```suggestion
- [GlobaliD Connect](./globalid-connect): Demonstrates how to log in using GlobaliD
- [Verifier Agent](./verifier-agent): Demonstrates Verifier flow for validating a user is 18+
```
